README.md

```diff
@@ -1,2 +1,81 @@
-# docker-mssql-backup
-Docker images for periodic backups from an Microsoft SQL Server container
+# Docker image for backup of Microsoft SQL Server
+
+Docker image to create regular backups of a [Microsoft SQL Server] image.
+
+## Information
+
+| Service | Stats                                                                                     |
+|---------|-------------------------------------------------------------------------------------------|
+| Docker  | [![Build](https://img.shields.io/docker/cloud/build/bbtsoftwareag/mssql-backup.svg?style=flat-square)](https://hub.docker.com/r/bbtsoftwareag/mssql-backup/builds) [![Pulls](https://img.shields.io/docker/pulls/bbtsoftwareag/mssql-backup.svg?style=flat-square)](https://hub.docker.com/r/bbtsoftwareag/mssql-backup) [![Stars](https://img.shields.io/docker/stars/bbtsoftwareag/mssql-backup.svg?style=flat-square)](https://hub.docker.com/r/bbtsoftwareag/mssql-backup) [![Automated](https://img.shields.io/docker/cloud/automated/bbtsoftwareag/mssql-backup.svg?style=flat-square)](https://hub.docker.com/r/bbtsoftwareag/mssql-backup/builds) |
+| GitHub  | [![Last commit](https://img.shields.io/github/last-commit/bbtsoftware/docker-mssql-backup.svg?style=flat-square)](https://github.com/bbtsoftware/docker-mssql-backup/commits/master) [![Issues](https://img.shields.io/github/issues-raw/bbtsoftware/docker-mssql-backup.svg?style=flat-square)](https://github.com/bbtsoftware/docker-mssql-backup/issues) [![PR](https://img.shields.io/github/issues-pr-raw/bbtsoftware/docker-mssql-backup.svg?style=flat-square)](https://github.com/bbtsoftware/docker-mssql-backup/pulls) [![Size](https://img.shields.io/github/repo-size/bbtsoftware/docker-mssql-backup.svg?style=flat-square)](https://github.com/bbtsoftware/docker-mssql-backup/) [![License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/bbtsoftware/docker-mssql-backup/blob/master/LICENSE) |
+
+## General
+
+| Topic  | Description                                                            |
+|--------|------------------------------------------------------------------------|
+| Image  | See [Docker Hub](https://hub.docker.com/r/bbtsoftwareag/mssql-backup). |
+| Source | See [GitHub](https://github.com/bbtsoftware/docker-mssql-backup).      |
+
+## Usage
+
+This container can create backups on a [Microsoft SQL Server] container.
+
+**NOTE:**
+The backup is written to a directory `/backup` inside the [Microsoft SQL Server] container, not to a volume in the backup container.
```

Comment:
Does that make sense?

Reply:
@eoehen Open to other ideas (but probably not in a first release due to time constraints)